CODEOWNERS

```diff
@@ -1,15 +1,13 @@
-# This file represents the ownership schema for the TTS Handbook. At the moment,
-# only Markdown files are included, but other file types may be added in the
-# future.
-# TODO how should GitHub teams be created for ownership?
+# This file, in addition to enforcing code ownership, represents the 
```

Comment:
Is "enforcing" correct? Since we don't require codeowner review as part of the protected branch configuration?

Reply:
Ah yes, this is a great point. I'll use a better word.